### PR TITLE
Fixes #14839: Use rudder agent check at postinst

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14839

### CODE REVIEW

File: rudder-agent/SOURCES/rudder-agent-postinst [hunk 1]
```diff
@@ -91,85 +84,18 @@ fi
 # Copy CFEngine binaries (Changed location from sbin/ to bin/ in version 3.4)
 cp -r -f /opt/rudder/bin/cf-* ${CFE_DIR}/bin/
 cp -r -f /opt/rudder/bin/rpmvercmp ${CFE_DIR}/bin/
-NB_COPIED_BINARIES=`ls -1 ${CFE_DIR}/bin/ | wc -l`
```
Comment (amousset) [inline rudder-agent/SOURCES/rudder-agent-postinst]: :man_facepalming:

File: rudder-agent/SOURCES/rudder-agent-postinst [hunk 2]
```diff
@@ -91,85 +84,18 @@ fi
...
-# Remove the lock on CFEngine
-if [ ${I_SET_THE_LOCK} -eq 1 ]; then
-  ${RUDDER_CMD} agent enable
-fi
 
 # Ensure the modification date of the capability file is correct
 # as it is used for cache invalidation in ncf lis-compatible-inputs
 touch /opt/rudder/etc/agent-capabilities
 
-# Restart daemons if we stopped them, otherwise not
```
Comment (amousset) [inline rudder-agent/SOURCES/rudder-agent-postinst]: Shouldn't we restart the service in postinst anyway?

Comment (peckpeck) [reply rudder-agent/SOURCES/rudder-agent-postinst]: i don't know, why should we ?

Comment (peckpeck) [reply rudder-agent/SOURCES/rudder-agent-postinst]: and the code below didn't restart but just started it

Comment (amousset) [reply rudder-agent/SOURCES/rudder-agent-postinst]: to use the new cf-serverd and cf-execd binaries at least I think.

Comment (peckpeck) [reply rudder-agent/SOURCES/rudder-agent-postinst]: what if we stop it and let agent-check restart it ?

Comment (amousset) [reply rudder-agent/SOURCES/rudder-agent-postinst]: possible too

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**